elasticsearch/Dockerfile

```diff
@@ -1,33 +1,43 @@
-FROM elasticsearch:5.6.16
+FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
 
 MAINTAINER OpenShift Development <dev@lists.openshift.redhat.com>
 
 EXPOSE 9200
 EXPOSE 9300
 USER 0
 
-ENV ES_CONF=/etc/elasticsearch/ \
+ENV ES_PATH_CONF=/etc/elasticsearch/ \
```

Comment:
nit but i'd suggest `ES_CONF_PATH` instead

Reply:
the elasticsearch scripts use `ES_PATH_CONF` - can't use anything else